pkg/proxy/userspace/proxysocket.go

```diff
@@ -73,7 +73,7 @@ func newProxySocket(protocol api.Protocol, ip net.IP, port int) (proxySocket, er
 }
 
 // How long we wait for a connection to a backend in seconds
-var endpointDialTimeout = []time.Duration{1, 2, 4, 8}
+var endpointDialTimeout = []time.Duration{250 * time.Millisecond, 500 * time.Millisecond, 2 * time.Second, 4 * time.Second}
```

Comment:
ah, I have been dealing with readtimeouts way more than I want to :)
this is dialtimeout :)
will update PR